### ISSUE TITLE
Example for default auth mode change for argo server

### ISSUE THREAD

**Issue by kabilan6** [argoproj/argo#2497]
can someone provide an example to change the default auth mode setting for argo server ?

**Comment by alexec** [comment]
```
argo server --auth-mode client
```

**Comment by kabilan6** [comment]
i tried to run the command from CLI and it throwed below error

 argo server --auth-mode server
INFO[0000]                                               authMode=server baseHRef=/ managedNamespace= namespace=argo-qa
WARN[0000] ConfigMap 'workflow-controller-configmap' does not have key 'config'
INFO[0000] Config Map:
WARN[0000] failed to listen: listen tcp :2746: bind: address already in use
WARN[0000] failed to listen: listen tcp :2746: bind: address already in use
WARN[0001] failed to listen: listen tcp :2746: bind: address already in use
WARN[0001] failed to listen: listen tcp :2746: bind: address already in use
WARN[0002] failed to listen: listen tcp :2746: bind: address already in use
ERRO[0002] timed out waiting for the condition

do we have equivalent kubectl command ?

**Comment by alexec** [comment]
you already have a process listening on the port, you will need to kill it

**Comment by kabilan6** [comment]
Thanks @alexec . Two questions 
1.  Can we set auth mode as part of argo server deployment ? what is the args that should be added/changed in the deployment yaml?
2. when auth-mode is set to client, i was able to submit the cron/workflow without passing the token in the argo UI. the submit options shouldn't be exposed when the token is not passed. I used the argo-server services endpoint ip:port address to launch the UI.

Argo Version - argo: vv2.5.2
kubernetes_version: 1.14.8
